mediaplatform_jwp/models.py

```diff
@@ -273,14 +273,15 @@ def get_sources(self):
     #: A property which calls get_sources and caches the result.
     sources = cached_property(get_sources, name='sources')
 
-    def fetch_size(self):
+    def get_resource(self):
```

Comment:
I think this should just be
```python
@cached_property
def resource(self):
    # ...
```
The ``sources`` property is a special case because sometimes one genuinely does want to re-call the API. With the ``resource`` prop, we're just doing database stuff so we should keep the same caching semantics as normal related fields.